### DISCUSSION TITLE
Problems with some terms in netavark

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [containers/netavark#629] [Q&A]
1. I can't really figure out why `--file` and `--config` both mention configuration, which is currently giving me a bit of trouble trying to add new parameters.
2. Why does the [firewall backend](https://www.cni.dev/plugins/current/meta/firewall/) in cni plugins become firewall driver in netavark?

**Comment by unknowndevQwQ** [comment]
@Luap99 I need your help.

**Comment by Luap99** [comment]
--config is just a path for the aardvark-dns configs

--file contains the actual information which is used to setup/teardown the networks

**Reply by unknowndevQwQ** [reply]
What about the answer to the other question?

**Reply by baude** [reply]
in netavark we call them drivers based on how we wrote the softrware.  CNI calls them backends.

**Reply by unknowndevQwQ** [reply]
Now I named the parameter --firewall-driver, but as a short parameter it conflicts with --file.
I want to rename --file to --profile.

**Reply by Luap99** [reply]
Just do not add a short option and it will work (i.e. remove the `short` from the clap macro), we should not rename existing things.
